packages/components/src/theme/grid.scss

```diff
@@ -368,13 +368,14 @@ $table-cell-padding: 4px 2px;
 
     .cell-warning {
       position: relative;
+      background-color: $panel-danger-heading-bg;
```

Comment:
This looks good in Chrome but I noticed in FF the red "corner" isn't being clipped:
<img width="562" alt="Screenshot 2024-07-10 at 7 59 47 AM" src="https://github.com/LabKey/labkey-ui-components/assets/6411206/411f3c8e-99c5-470f-a7fc-15f34fad07c8">

Reply:
Should be better now